Scripts/Python/stupStartUp.py

```diff
@@ -104,3 +108,20 @@ def OnServerInitComplete(self):
             PtShowDialog("GUIDialog04b")
         else:
             PtShowDialog("GUIDialog06")
+
+        self.randomizeBackground()
+
+    
+    ###########################
+    def randomizeBackground(self):
+        if not BGObj.sceneobject or not BGLayer.layer:
+            PtDebugPrint("stupStartUp: Missing ptAttribs for randomized backgrounds.  Leaving default...") 
+            return
+
+        # Get the first available ImageLibModifier on our Background SceneObject
+        if ilm := BGObj.sceneobject.getImageLibMods()[0]:
```

Comment:
I don't think this will gate properly if there are no image lib mods. You'll be indexing into an empty sequence. This is probably more robust:


```suggestion
        if ilm := next(BGObj.sceneobject.getImageLibMods(), None):
```

Reply:
Good idea.  Done!